src/tools/Tools.mjs

```diff
@@ -15,134 +14,30 @@ export default class Tools {
         }
 
         let factory = (cb, stage) => {
-            cb(null, this.createRoundRect(stage, w, h, radius, strokeWidth, strokeColor, fill, fillColor));
-        }
+            stage.platform.createRoundRect(cb, stage, w, h, radius, strokeWidth, strokeColor, fill, fillColor);
+        };
         let id = 'rect' + [w, h, strokeWidth, strokeColor, fill ? 1 : 0, fillColor].concat(radius).join(",");
         return Tools.getCanvasTexture(factory, id);
     }
 
-    static createRoundRect(stage, w, h, radius, strokeWidth, strokeColor, fill, fillColor) {
```

Comment:
This function exposed to framework's public API and cannot be removed or changed freely. If you want to change implementation, you need change original function to be adapter, e.g.

```javascript
static createRoundRect(stage, w, h, radius, strokeWidth, strokeColor, fill, fillColor) {
    stage.platform.createRoundRect(null, stage, w, h, radius, strokeWidth, strogeColor, fill, fillColor);
}
```

Reply:
Updated the code based on your feedback and suggestion.